exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/ParquetBooleanPredicate.java

```diff
@@ -27,8 +27,7 @@
 /**
  * Boolean predicates for parquet filter pushdown.
  */
-public abstract class ParquetBooleanPredicate<C extends Comparable<C>> extends BooleanOperator
-    implements ParquetFilterPredicate<C> {
+public abstract class ParquetBooleanPredicate<C extends Comparable<C>> extends BooleanOperator implements ParquetFilterPredicate<C> {
```

Comment:
avoid format only changes unless something is wrong with the way how the code is formatted.

Reply:
Completely agreed.
I just applled the intellij-idea-settings.jar which is provided by DRILL team.
As you may notice, I didn't apply it to the other files that are modified in this PR, to avoid having too many changes which are not meaningful.

Reply:
That `intellij-idea-settings.jar` should be applied to the part of the code that is modified, not to the entire file or other files. Please revert back all format only changes.

Reply:
Done